TAC/Meetings/2022/2022-02-24/CCC_TAC_Minutes-2022-02-24.md

```diff
@@ -0,0 +1,193 @@
+# Confidential Computing Consortium TAC meeting
+
+*The TAC meets for two hours on alternating Thursdays. All members of the community are welcome to attend and participate.*
+
+## Meeting details
+
+**Date**: Thursday, February 24, 2022
+
+**Time**: 7a-9a US Pacific Time
+
+* **Zoom**: [https://zoom.us/j/184384055](https://zoom.us/j/184384055) (pw: 43690)
+* **Calendar**: [https://calendar.confidentialcomputing.io](https://calendar.confidentialcomputing.io) ([ical](https://calendar.google.com/calendar/ical/c_c0pcihr7n2n1k3a38i32d9ag10%40group.calendar.google.com/public/basic.ics), [Google Calendar](https://calendar.google.com/calendar/u/0/r?cid=c_c0pcihr7n2n1k3a38i32d9ag10@group.calendar.google.com))
+* **Recording**: [YouTube](https://www.youtube.com/watch?v=cAKU9u0Z0_g)
+
+### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* **Accenture** - Giuseppe Giordano
+* **ANT Group** - Zongmin Gu
+* **ARM** - Thomas Fossati / Michael Lu
+* **Google** - Iulia Ion
+* **Huawei** - Zhipeng (Howard) Huang
+* **Intel** - Dan Middleton / Simon Johnson
+* **Meta** - Eric Northup / Shankaran Gnanashanmugam
+* **Microsoft** - Dave Thaler (TAC chair)
+* **Red Hat** - Lily Sturmann / Dimitrios Pendarakis
+
+## Links
+
+* **Code of Conduct**: [code-of-conduct.confidentialcomputing.io](https://code-of-conduct.confidentialcomputing.io)
+* **CCC Charter**: [charter.confidentialcomputing.io](https://charter.confidentialcomputing.io)
+* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+
+Rep. Thaler opened the call at 7:04a PT. He welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation.
+
+### Roll Call / Attendance
+
+#### Voting members
+
+* Dan Middleton (Intel)
+* Dave Thaler (Microsoft)
+* Lily Sturman (Red Hat)
+* Giuseppe Giordano (Accenture)
+* Tate, Hongliang Tian (Ant Group)
```

Comment:
Unless Tate or Zongmin tell us otherwise in a future meeting, let's add Tate to the list of alternates for Ant. I'm also going to push a fix changing "ANT" to "Ant."